src/v/cluster/data_migration_frontend.cc

```diff
@@ -366,13 +366,15 @@ ss::future<std::error_code> frontend::do_remove_migration(id id) {
     validate_migration_shard();
     auto ec = co_await insert_barrier();
     if (ec) {
+        vlog(dm_log.warn, "failed waiting for barrier: {}", ec);
```

Comment:
the commit message states that the logging is being added in the topics table but here we add it in the `data_migrations` frontend